source/sorted_table_model.ts

```diff
@@ -21,22 +21,32 @@ export class ColumnOrder {
    * @param sortOrder - Whether the column is ordered in ascending or descending
    *        order.
    */
-  constructor(index: number, sortOrder: SortOrder = SortOrder.ASCENDING) {}
+  constructor(index: number, sortOrder: SortOrder = SortOrder.ASCENDING) {
+    this.columnIndex = index;
```

Comment:
For situations like this, prefix the member variable/field with an underscore, ie. ```this._index = index```.